src/equations/compressible_euler_1d.jl

```diff
@@ -497,6 +497,87 @@ function flux_hll(u_ll, u_rr, orientation, equations::CompressibleEulerEquations
   return SVector(f1, f2, f3)
 end
 
+ """                                                                                                
+    flux_hllc(u_ll, u_rr, orientation, equations::CompressibleEulerEquations2D)                     
```

Comment:
Ok, sorry. No idea how this happend. Will check all files.

Reply:
Just a guess... did you copy the 2D implementations to 1D and 3D from within `vim` using your mouse? Because that has the same effect of adding unwanted trailing newlines on my computer...

Reply:
yeah, maybe...no clue, it was in many more files as well...I corrected it now.